### PR TITLE
Remove noImplicitAny

### PR DESCRIPTION
**Pull request by jnatalzia**
#### Addresses Issue(s):

JW8-###

### Checklist
- [ ] Jenkins builds and unit tests are passing
- [ ] I have reviewed the automated results

### CODE REVIEW

**Comment by jnatalzia** [comment]
test this please

File: src/js/utils/parser.ts [hunk 1]
```diff
@@ -26,7 +26,7 @@ export function getAbsolutePath(path: string, base?: string): string {
         basepath = basepath.substring(protocol.length + domain.length + 1, basepath.lastIndexOf('/'));
         patharray = basepath.split('/').concat(path.split('/'));
     }
-    const result = [];
+    const result: string[] = [];
     for (let i = 0; i < patharray.length; i++) {
```
Comment (mamaddox) [inline src/js/utils/parser.ts]: Interesting that these failed when removing noImplicitAny

Comment (jnatalzia) [reply src/js/utils/parser.ts]: yeah right? not quite clear on that - makes me feel more secure in removing it given it seems like there's some phantom behavior or something

Comment (mamaddox) [reply src/js/utils/parser.ts]: definitely agree

**Pull request merged**